polling_stations/apps/data_importers/management/commands/teardown.py

```diff
@@ -59,26 +87,11 @@ def handle(self, *args, **kwargs):
 
         if kwargs["council"]:
             for council_id in kwargs["council"]:
-                print("Deleting data for council %s..." % (council_id))
-                # check this council exists
-                council_obj = Council.objects.get(pk=council_id)
-                gss_code = council_obj.geography.gss
-
-                PollingStation.objects.filter(council=council_id).delete()
-                PollingDistrict.objects.filter(council=council_id).delete()
-                AdvanceVotingStation.objects.filter(council=council_id).delete()
-
-                UprnToCouncil.objects.filter(lad=gss_code).update(polling_station_id="")
-
-                dq = DataQuality.objects.get(council_id=council_id)
-                dq.report = ""
-                dq.num_addresses = 0
-                dq.num_districts = 0
-                dq.num_stations = 0
-                dq.save()
-
-                record_teardown_event(council_id)
-                print("..done")
+                self.teardown_council(council_id)
+
+        elif kwargs.get("eoni"):
+            for council_id in NIR_IDS:
+                self.teardown_council(council_id)
```

Comment:
In the help for the `--council` flag it says

`Council ID to clear in the format X01000001`

but `NIR_IDS` is a list of 3 letter codes: ABC, AND, ANN...

This seems like something is slightly amiss as we are passing the same thing into `teardown_council` either way. Have we modified teardown so that `--council` now expects a 3 letter code? If so, lets update the help.

Reply:
Good shout. 
This was a side effect of starting to use reg codes as council ids, and moving gss codes to the `identifiers` field on the council geography model.